ipod-shuffle-4g.py

```diff
@@ -364,6 +371,11 @@ def populate(self, filename):
         if os.path.splitext(filename)[1].lower() in (".m4a", ".m4b", ".m4p", ".aa"):
             self["filetype"] = 2
 
+        if "/iPod_Control/Podcasts/" in filename:
+            self.is_podcast = True
+            self["dontskip"] = 0
+            self["remember"] = 1
```

Comment:
Could you please add more explanation what this does?

Reply:
update incoming